Application/EdFi.Ods.Api/Controllers/DataManagementControllerBase.cs

```diff
@@ -213,6 +221,7 @@ public virtual async Task<IActionResult> Get(Guid id)
         [ProducesResponseType(StatusCodes.Status204NoContent)]
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status412PreconditionFailed)]
+        [Produces(MediaTypeNames.Application.Json)]
```

Comment:
Is there a way to set this globally, since the majority, if not all responses are return in json?

Reply:
yes however in this case, change only get and get all depending on the profile if applied. Otherwise to set globally we attach the attribute to the controller